<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Serial Tool</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="7">
      <widget class="QComboBox" name="cob_UpDown">
       <item>
        <property name="text">
         <string>Up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Down</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QPushButton" name="pb_MakeStation">
       <property name="text">
        <string>MakeStation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb_OpenOrClose">
       <property name="text">
        <string>打开(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cob_Com"/>
     </item>
     <item row="1" column="5">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item row="2" column="9">
      <widget class="QCheckBox" name="chk_HexSend">
       <property name="text">
        <string>Hex Send</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QPushButton" name="pb_Send">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Send(&amp;S)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QSpinBox" name="sp_CircleT"/>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="chk_HexRecv">
       <property name="text">
        <string>Hex Recv</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QSpinBox" name="sp_Station"/>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pb_ClearRecv">
       <property name="text">
        <string>Clear Recv</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_Com">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="10">
      <widget class="QTextEdit" name="te_Recv"/>
     </item>
     <item row="2" column="0" rowspan="3" colspan="9">
      <widget class="QTextEdit" name="te_Send"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_Status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
